cmd/wallet-web/src/components/InputField/InputField.vue

```diff
@@ -19,7 +18,7 @@
     <div class="fader" />
     <!-- TODO: use inline svg instead once https://github.com/trustbloc/edge-agent/issues/816 is fixed -->
     <img src="@/assets/img/danger-icon.svg" />
-    <label class="input-wordlimit">{{ characterCount }}</label>
+    <span>{{ characterCount }}</span>
```

Comment:
you removed the class this will impact the ui placing of the character count

Reply:
My bad, it's back now